### PR TITLE
Changed ping() description

### PR DESCRIPTION
**Pull request by owlishDeveloper**
This is my fix for bug https://bugzilla.mozilla.org/show_bug.cgi?id=1262612 .

### CODE REVIEW

**Comment by owlishDeveloper** [comment]
@djmitche r?

File: src/v1.js [hunk 1]
```diff
@@ -880,9 +880,8 @@ api.declare({
   stability:  'experimental',
```
Comment (djmitche) [inline src/v1.js]: Let's change this to 'stable' too

**Comment by jonasfj** [comment]
so wouldn't it be much better if we implemented the ping method in:
https://github.com/taskcluster/taskcluster-lib-api

When we would have the ping method in all systems... and we have the same implemetation everywhere...

**Comment by owlishDeveloper** [comment]
@djmitche r?

**Comment by djmitche** [comment]
deploy'd to production

**Pull request merged**